### PR TITLE
Add qie11 task 81x

### PR DESCRIPTION
**Pull request by deguio**
add QIE11 task to the hcal calibration application.
same as #16141

### CODE REVIEW

**Comment by kpedro88** [comment]
@cmsbuild please test

**Comment by kpedro88** [comment]
@cmsbuild please test

**Comment by dmitrijus** [comment]
+1

File: DQM/HcalTasks/plugins/QIE11Task.cc [hunk 1]
```diff
@@ -134,6 +140,32 @@ QIE11Task::QIE11Task(edm::ParameterSet const& ps):
 	}
 }
 
+
+/* virtual */ bool QIE11Task::_isApplicable(edm::Event const& e)
+{
+  if (_ptype!=fOnline || _laserType < 0)
+    return true;
+  else
```
Comment (dmitrijus) [inline DQM/HcalTasks/plugins/QIE11Task.cc]: if you don't use {} for the first block, don't use it for the else-block either. It looks rather confusing....

**Comment by davidlange6** [comment]
+1

**Pull request merged**